sw-precache-config.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  staticFileGlobs: [
+    'out/*.html',
+    'out/static/**.*',
```

Comment:
Image files are not properly pre-cached.

Please change to this: `out/static/**/*`.